OpenAI.Playground/App.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Runtime.CompilerServices.Unsafe" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
```

Comment:
why do we need this package?

Reply:
System.Runtime.CompilerServices.Unsafe is a .Net Standard 2 dependency for other packages. There is a package version incompatibility however with System.Memory requiring version 4.5.4 and the others requiring version 6. So I have put in the binding redirect to solve it.